Extension error (sphinxcontrib.newsfeed)!

### Describe the bug

After updating to sphinx 9.1 (and the requirements sphinx_rtd_theme==3.1.0 & myst_parser==5.0.0), I get:

```
Extension error (sphinxcontrib.newsfeed)!

Versions
========

* Platform:         linux; (Linux-6.11.0-1018-azure-x86_64-with-glibc2.39)
* Python version:   3.13.11 (CPython)
* Sphinx version:   9.1.0
* Docutils version: 0.22.4
* Jinja2 version:   3.1.6
* Pygments version: 2.19.2

Last Messages
=============



    writing output... [ 90%]
    news/20251125-release-2026


    writing output... [ 91%]
    news/news

    sphinx-sitemap: sitemap.xml was generated for URL https://zeiss.github.io/zeiss-inspect-app-api/main/ in /home/runner/work/zeiss-inspect-app-api/zeiss-inspect-app-api/_build/sitemap.xml

Loaded Extensions
=================

* sphinx.ext.mathjax (9.1.0)
* alabaster (1.0.0)
* sphinxcontrib.applehelp (2.0.0)
* sphinxcontrib.devhelp (2.0.0)
* sphinxcontrib.htmlhelp (2.1.0)
* sphinxcontrib.serializinghtml (2.0.0)
* sphinxcontrib.qthelp (2.0.0)
* myst_parser (5.0.0)
* sphinxcontrib.jquery (4.1)
* sphinx_rtd_theme (unknown version)
* sphinx_favicon (unknown version)
* sphinx.ext.githubpages (9.1.0)
* sphinx_sitemap (2.9.0)
* sphinxcontrib.newsfeed (unknown version)

Traceback
=========

      File "/opt/hostedtoolcache/Python/3.13.11/x64/lib/python3.13/site-packages/sphinx/events.py", line 452, in emit
        raise ExtensionError(
        ...<3 lines>...
        ) from exc
    sphinx.errors.ExtensionError: Handler <function process_feed at 0x7fcbd9d51da0> for event 'doctree-resolved' threw an exception (exception: 'body')


The full traceback has been saved in:
/tmp/sphinx-err-ji2h65yv.log

To report this error to the developers, please open an issue at <https://github.com/sphinx-doc/sphinx/issues/>. Thanks!
Please also report this if it was a user error, so that a better error message can be provided next time.
Error: Process completed with exit code 2.
```

### How to Reproduce

See https://github.com/ZEISS/zeiss-inspect-app-api/actions/runs/21211960846/job/61022575787

--> `conf.py`

```
# Configuration file for the Sphinx documentation builder.
#
# For the full list of built-in configuration values, see the documentation:
# https://www.sphinx-doc.org/en/master/usage/configuration.html

# -- Project information -----------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information

project = 'App Development Documentation'
copyright = '2025, Carl Zeiss GOM Metrology GmbH'
author = 'Carl Zeiss GOM Metrology GmbH'

# -- General configuration ---------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration

extensions = ['myst_parser', 'sphinx_rtd_theme', 'sphinx_favicon', 'sphinx.ext.githubpages', 'sphinx_sitemap', 'sphinxcontrib.newsfeed']
#extensions = ['myst_parser', 'sphinx_rtd_theme', 'sphinx_favicon', 'sphinx.ext.githubpages', 'sphinx_sitemap']
source_suffix = ['.rst', '.md']

templates_path = ['_templates']
exclude_patterns = ['README.md']

myst_enable_extensions = [
  'colon_fence',
  'fieldlist',
  'replacements',
  'deflist',
  'attrs_inline'
]

myst_heading_anchors = 4

# Suppress the warning "WARNING: Non-consecutive header level increase" which commonly appears in python_api/python_api.md
# Unfortunately, suppressing this only for a single file does not seem to work...
suppress_warnings  = ['myst.header']

# -- Options for Jupyter Notebook --------------------------------------------
jupyter_execute_notebooks = "off"

# -- Options for sitemap -----------------------------------------------------
# https://sphinx-sitemap.readthedocs.io/en/latest/getting-started.html
html_baseurl = 'https://zeiss.github.io/zeiss-inspect-app-api/main/'
sitemap_url_scheme = "{link}"

# -- Options for HTML output -------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
# Do not show "View page source" link on every page
html_show_sourcelink = False

html_theme = "sphinx_rtd_theme"
html_theme_options = {
  # Toc options
  'includehidden': False
}

# "sphinx_rtd_theme" appends " &mdash; <project>  documentation" to the page heading -
# this changes the title to "<page_heading> &mdash; <project>"
html_title = project
html_static_path = ['_static']

# -- Override some "sphinx_rtd_theme" styles to match ZEISS branding ---------------
# https://docs.readthedocs.io/en/stable/guides/adding-custom-css.html
html_style = "css/theme_zeiss.css"

favicons = [
    {
        "rel": "icon",
        "sizes": "16x16",
        "static-file": "favicon.png",  # => use `_static/favicon.png`
        "type": "image/png",
    }
]

# Source: https://brand.zeiss.com/cmsPublic/brandportal/basic-design-elements/logo-tagline.html
html_logo =  "_static/zeiss-logo-rgb.png"
```

--> `doc/news/20251125-release-2026.md` (File containing `feed-entry` directive)

---
# ZEISS INSPECT 2026 Released

```{eval-rst}
.. feed-entry::
   :date: 2025-11-25 11:00
```

We are happy to announce that [ZEISS INSPECT 2026](https://www.zeiss.com/metrology/en/software/zeiss-inspect.html) has been released. Watch out for App development related enhancements and new features!
---

--> `news.md` (File containing `feed`-directive)

---
# What's New

<a href="../index.rss">&#x200B;<i class="fa fa-rss-square"></i> Subscribe to RSS News Feed</a>

```{eval-rst}
.. feed::
   :rss: index.rss
   :title: ZEISS INSPECT 2026 App Development Documentation News
   :link: https://zeiss.github.io/zeiss-inspect-app-api/2026/

   20251125-release-2026
```
---

### Environment Information

```text
Platform:              linux; (Linux-6.11.0-1018-azure-x86_64-with-glibc2.39)
Python version:        3.13.11 (main, Dec  8 2025, 02:51:34) [GCC 13.3.0])
Python implementation: CPython
Sphinx version:        9.1.0
Docutils version:      0.22.4
Jinja2 version:        3.1.6
Pygments version:      2.19.2
```

### Sphinx extensions

```python
['myst_parser', 'sphinx_rtd_theme', 'sphinx_favicon', 'sphinx.ext.githubpages', 'sphinx_sitemap', 'sphinxcontrib.newsfeed']
```

### Additional context

_No response_

--- a/doc/changes/9.0.rst
+++ b/doc/changes/9.0.rst
@@ -71,7 +71,8 @@ Incompatible changes
   Patches by Adam Turner.
 * #13355: Don't include escaped title content in the search index.
   Patch by Will Lachance.
-* #13664: HTML builder: :meth:`!sphinx.builders.html.StandaloneHTMLBuilder.render_partial`
+* #13664: HTML builder:
+  :meth:`!sphinx.builders.html.StandaloneHTMLBuilder.render_partial`
   now returns ``fragment`` and ``title`` keys only.
   Patch by Adam Turner.
 

--- a/doc/changes/9.0.rst
+++ b/doc/changes/9.0.rst
@@ -71,6 +71,9 @@ Incompatible changes
   Patches by Adam Turner.
 * #13355: Don't include escaped title content in the search index.
   Patch by Will Lachance.
+* #13664: HTML builder: :meth:`!sphinx.builders.html.StandaloneHTMLBuilder.render_partial`
+  now returns ``fragment`` and ``title`` keys only.
+  Patch by Adam Turner.
 
 Deprecated
 ----------

